tests/test_utilities.py

```diff
@@ -1042,6 +1042,68 @@ def test_parse_datetime_handle_invalid(self, datetime_value, expected_parsed_dat
         """
         assert utils.parse_datetime_handle_invalid(datetime_value) == expected_parsed_datetime
 
+    @ddt.data(
+        (
+            #advertised course run is set and included, return that one
```

Comment:
to get quality CI check to pass, you need to change these comments to have a space after the hashmark:

`E265 block comment should start with '# '`